app/models/order.rb

```diff
@@ -0,0 +1,13 @@
+class Order < ApplicationRecord
+  enum status: {pending: 0, accepted: 1, cancel: 2, paid: 3}
```

Comment:
sao lại dùng kiểu hash mà không phải là như này?
```suggestion
  enum status: %i(pending accepted cancel paid)
```

Reply:
để kiểu này lúc mình có thay đổi thứ tự enum thì dễ hơn, kiểu cho pending thành 3 thì thay mỗi 0 thành 3 là ok